README.md

```diff
@@ -17,27 +17,19 @@ You can download the binary built on master branch from [<img src="https://dply.
 * Show Information Feed
 
 # Contributing
-We are always welcome your contribution!
+
+We are always welcome your contribution! See [CONTRIBUTING.md](CONTRIBUTING.md)
```

Comment:
We welcome any and all contributions! See [CONTRIBUTING.md](CONTRIBUTING.md) for more information